src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/ProviderName.jsx

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default function ProviderName({ appointment, useV2 = false }) {
-  const { name, practiceName, providerName } =
```

Comment:
Removed `name` and `providerName` since these are attributes used in the Scheduling Manager services which is decommissioned in favor of VAOS service.

Reply:
I think this was supposed to be removed `name` + `practice name`